lib/capistrano/slack_notify.rb

```diff
@@ -91,8 +91,12 @@ def destination
       fetch(:slack_destination, stage)
     end
 
+    def repository
+      fetch(:repository, 'origin')
```

Comment:
It's my understanding (might be completely wrong) that :repository would be the capistrano's config for the remote's url?